shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -19,6 +19,20 @@
 #import "flutter/shell/platform/darwin/ios/ios_surface.h"
 #import "flutter/shell/platform/darwin/ios/ios_surface_gl.h"
 
+@implementation UIView (FirstResponder)
+- (BOOL)hasFirstResponderInViewHierarchySubtree {
```

Comment:
Another way is to make this an instance method of `FlutterTouchInterceptingView`, to avoid categories.
I don't have preference though